lib/storage/client.js

```diff
@@ -1196,6 +1196,42 @@ class Client extends CryptoConsumer {
     }
     return returnValues
   }
+  /**
+   *
+   * @param {string} clientID   UUID assigned by TozStore, used to identify the client; optional.
+   * @param {String} groupName Group name you want to get information regarding.
+   *
+   * @return {Object}  The group object or null if it does not exist.
+   */
+  async groupInfo(clientID = null, groupName) {
```

Comment:
nit: reverse parameters so you can call `groupInfo('groupName') without the need to set clientID.

Also: default value of non-passed arguments is `undefined` so you can probably omit the null value and just check for a falsey value.